src/status_im/chat/models/input.cljs

```diff
@@ -184,9 +184,11 @@
 
 (defn build-image-messages
   [{db :db} chat-id]
-  (let [images (get-in db [:chat/inputs chat-id :metadata :sending-image])]
+  (let [images (get-in db [:chat/inputs chat-id :metadata :sending-image])
+        album-id (str (random-uuid))]
```

Comment:
could you elaborate why we generate it here? and also str is not needed? its already string i believe

Reply:
@flexsurfer Images that are sent together we need to give them the same `album-id`.
`random-uuid` return some object. I have to use `str` to convert it to string.